pkg/cloudclient/cloudclient.go

```diff
@@ -3,18 +3,48 @@ package cloudclient
 import (
 	"context"
 	"fmt"
+	"os"
 	"time"
 
-	awscredsv2 "github.com/aws/aws-sdk-go-v2/credentials"
-	awscredsv1 "github.com/aws/aws-sdk-go/aws/credentials"
 	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
 	awsCloudClient "github.com/openshift/osd-network-verifier/pkg/cloudclient/aws"
-	gcpCloudClient "github.com/openshift/osd-network-verifier/pkg/cloudclient/gcp"
 	"github.com/openshift/osd-network-verifier/pkg/output"
+)
+
+const GCP = "GCP"
+const AWS = "AWS"
+
+// common commandline args
+type CmdOptions struct {
+	CloudType    string // not required. if provided, currently only supports "aws".
+	CloudTags    map[string]string
+	Debug        bool
+	Region       string
+	AwsProfile   string
+	InstanceType string
+	VpcSubnetID  string
+	CloudImageID string
+	Timeout      time.Duration
+	KmsKeyID     string
+}
 
-	"golang.org/x/oauth2/google"
+var (
+	DefaultTags     = map[string]string{"osd-network-verifier": "owned", "red-hat-managed": "true", "Name": "osd-network-verifier"}
+	RegionEnvVarStr = "AWS_REGION"
+	RegionDefault   = "us-east-1"
+	DefaultTime     = 2 * time.Second
```

Comment:
These are AWS specific, and likely won't work for GCP

Reply:
Those are optional. But that's as intended. There's a card which I'm working on - to separate AWS and GCP functionality and make it agnostic. We should go deeper on that card, imo.

Reply:
If we're adding these and they're AWS specific, then they should exist in the AWS specific part of the cloudclient.

Reply:
I wrote an explanation https://github.com/openshift/osd-network-verifier/pull/107#issuecomment-1171778922